libs/scripts/scripts.js

```diff
@@ -161,6 +161,9 @@ const eagerLoad = (img) => {
 }());
 
 (async function loadPage() {
+  if (document.head.querySelector('meta[name="template"]')?.content === '404') {
```

Comment:
I'm not familiar with the load sequence on milo, but it is important that this happens before the first call to `sampleRUM`

Reply:
Yes this is the first thing that's called, only downside is it needs to be implemented by every consumer. 
Which might be a good thing, since we only need to increase 404 tracking for BACOM for now - so that particular project can implement it!

Reply:
Then, PR looks good to me

Reply:
This file already includes methods from `utils.js`, could we not use `getMetadata` for consistency?

On another note, we might need to help/reach out to create this PR on BACOM's side as well.

Reply:
Yup, will open a PR on the BACOM side of things!